recipes/crypto-createCredentials/codemod.yaml

```
@@ -0,0 +1,21 @@
+schema_version: "1.0"
+name: "@nodejs/crypto-createCredentials"
```

```suggestion
name: "@nodejs/crypto-create-credentials"
```

@AugustinMauroy I think we are using only kebab-case for codemod recipes names, right? 

I think we're using only kebab-case for codemod recipe names, right?

It would be great to have a section in `CONTRIBUTE.md` about how to name codemod recipes and the naming convention as well.

---

@0hmX just to you know this name `crypto-create-credentials` matches the recipe folder name, so you could you please update the folder name as well? cheers

---

Sure, I was also thinking the same